build.py

```diff
@@ -124,9 +124,30 @@ def run(config, start_class_path, arguments):
 ### def run ###
 
 
+# USAGE
+#
+# Print basic usage info.
+#
+def usage(args) :
+  print('Unknown command "%s"' % args)
+  print('Usage: ./build.py clean | build | rebuild | run')
+  print('  clean   : Remove all files in the output directory.')
+  print('            This does not remove the root of the output tree.')
+  print('  build   : Build the full project. This will build all java files')
+  print('            found in all of the src directories.')
+  print('  rebuild : perform clean followed by build.')
+  print('  run <class path> [ arguments ... ] : Run the specified class.')
+  print('            All arguments after the class path will be passed to')
+  print('            the java class when it runs.')
+
+
 # MAIN
 def main(args) :
```

Comment:
Well, actually I print a message if no parameters are provided.  I also am more informative if the run command is missing parameters.